### PR TITLE
Convert glean `event` schema from array to object

### PR DESCRIPTION
**Pull request by travis79**
Checklist for reviewer:

- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/include/glean/event.1.schema.json [hunk 1]
```diff
@@ -1,17 +1,16 @@
 {
-  "type": "array",
-  "items": [
-    {
-      "type": "integer",
-      "minimum": 0
+  "type": "object",
+  "properties": {
```
Comment (mdboom) [inline templates/include/glean/event.1.schema.json]: Let's add `"additionalProperties": false` here so these objects can't contain anything extra.  If we add something in the future, we'll want to add that explicitly and I *think* having something additional should be treated as an error.

But I also defer to @fbertsch on that question.

Comment (travis79) [reply templates/include/glean/event.1.schema.json]: `"additionalProperties": false` added

File: templates/include/glean/event.1.schema.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  "type": "array",
-  "items": [
-    {
-      "type": "integer",
-      "minimum": 0
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "timestamp": {
+      "type": "integer"
```
Comment (fbertsch) [inline templates/include/glean/event.1.schema.json]: We seem to have lost the `"minimum": 0` on `timestamp`

Comment (travis79) [reply templates/include/glean/event.1.schema.json]: Done!

**Pull request merged**